api/user_role/schema.py

```diff
@@ -19,6 +20,10 @@ class Arguments:
     user_role = graphene.Field(UsersRole)
 
     def mutate(self, info, **kwargs):
+        roles = UserRoleModel.query.filter_by(user_id=kwargs['user_id'],
```

Comment:
Hello, this query checks if a users already has roles and throws errors if we add more roles right, which should not be the case coz a user can ideally have multiple roles, e.g can be ops associate and admin, so if one had been added, another role can still be added, thought it was to prevent having same role twice for one user

Reply:
hey @shemogumbe, actually it does not throw an error if we add more roles to the user, a user can have multiple roles,  it only throws an error if we try to create the same user with the same role more than once ie user_1 with role_1 can only be created once but we can create  user_1 with role_2 which should be the case

Reply:
currently, we can create use_1 with role_1 severally and it duplicates itself on the database